docs/examples/lombok/Makefile

```
@@ -1,22 +1,24 @@
-ifeq (${JAVA_VER},26)
 all:
+ifeq (${JAVA_VER},26)
 	@echo "Skipping test because lombok does not work under Java ${JAVA_VER}"
 else
-all: clean
 	- ../../../gradlew build > Out.txt 2>&1
 	(grep -qF "User.java:9: error: [argument] incompatible argument for parameter y of FooBuilder.y." Out.txt \
 	  && grep -qF "Foo.java:12: error: [assignment] incompatible types in assignment." Out.txt \
 	  && grep -qF "y = null; // error" Out.txt) \
 	 || (echo "===== start of Out.txt =====" && cat Out.txt && echo "===== end of Out.txt =====" && false)
+all: clean
```

checkmake (0.3.2)

[warning] Target "all" defined multiple times (lines 8 and 17).

(uniquetargets)